src/backend/vulkan/src/lib.rs

```diff
@@ -511,6 +511,11 @@ impl hal::PhysicalDevice<Backend> for PhysicalDevice {
     }
 
     fn features(&self) -> Features {
+        // see https://github.com/gfx-rs/gfx/issues/1930
+        let is_windows_intel_kaby = cfg!(windows) &&
+            self.properties.vendor_id == 0x8086 &&
```

Comment:
Could you declare them as constants somewhere? Might be helpful in case we need more of these exceptions.